core/controllers/payload_validator.py

```diff
@@ -118,6 +118,7 @@ def convert_string_to_bool(param: Text) -> Optional[Union[bool, Text]]:
 # maintained. Please remove the name of the handlers if they already
 # contains schema.
 HANDLER_CLASS_NAMES_WHICH_STILL_NEED_SCHEMAS = [
+    'AllTopicNamesHandler',
```

Comment:
Please do not add anything here. Add the schemas to the handler.

Reply:
Done

Reply:
Not done?

Reply:
I'm sorry, I added the schema but forgot to remove AllTopicNamesHandler from the list. PTAL again.